tools/hyphy/hyphy_gard.xml

```diff
@@ -15,8 +15,8 @@
             #elif str($datatype.value) == 'amino-acid':
                 --model '$datatype.model'
             #end if
-            --rv '$rate_cond.rate'
-            #if str($rate_cond.rate) != 'None':
+            #if str($rate_cond.rate):
```

Comment:
is this one correct? Without this change, you will get `--rv 'None'` on the command line which looks not correct.

Reply:
That **will** be accepted by HyPhy, since `None` is a valid option

Reply:
I did some testing locally, and it looks like GARD expects either 'None', the other options, or the parameter not to be specified at all, while specifying `--rv=''` causes an error. I think this one is safe to leave as updated.

Reply:
That is correct. An explicit empty string will cause an abnormal (error report) exit, since it is not a valid option. 

```
$hyphy gard --help                                                                  

Available analysis command line options
---------------------------------------
Use --option VALUE syntax to invoke
If a [reqired] option is not provided on the command line, the analysis will prompt for its value
[conditionally required] options may or not be required based on the values of other options

type
	The type of data to perform screening on
	default value: nucleotide

code
	Genetic code to use (for codon alignments)
	default value: Universal
	applies to: Choose Genetic Code

alignment [required]
	Sequence alignment to screen for recombination

model
	The substitution model to use
	default value: JTT

rv
	Site to site rate variation
	default value: None

rate-classes
	How many site rate classes to use
	default value: 4

output
	Write the resulting JSON to this file (default is to save to the same path as the alignment file + 'GARD.json')
	default value: gard.defaultJsonFilePath [computed at run time]

output-lf
	Write the best fitting HyPhy analysis snapshot to (default is to save to the same path as the alignment file + 'best-gard')
	default value: gard.defaultFitFilePath [computed at run time]


```